src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +45,6 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream().filter(n -> n > 3).map(n -> n*2).reduce(0,Integer::sum);
```

Comment:
```suggestion
        return numbers.stream()
                .filter(n -> n > 3)
                .map(n -> n * 2)
                .reduce(0, Integer::sum);
```
- stream api는 operation 별로 개행해주는게 가독성에도 좋아 휴먼에러를 방지할 수 있을거 같아요.
- code format은 습관화해주세요~